### ISSUE TITLE
Compressive findings

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9931]
**Preferred term label:**
History of compressive footwear with synonyms Tight shoes, toe-box shoes, high heel shoes
History of compressive dressings with synonyms Tight bandages, compression sleeve, casts, splints, immobilizers, tight clothes, watch bands, handcuffs
**Synonyms**
As above

**Definition (free text, please give PubMed ID)**
We are keeping these 2 separate because we think patients see these differently and they would be elicited differently on history.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Note that we previously added
HP:6000302
Term Name: History of compressive footwear

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000682
- label: History of compressive dressings
- parents: [Past medical history (HP:0032443)](https://hpo.jax.org/app/browse/term/HP:0032443)
- pmid: 
- def: A past medical history of having been treated with tight bandages, compression sleeve, tight casts or splints, immobilizers, or similar items such as overly tight clothes, watch bands, or handcuffs.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9931
- synonyms:

**Comment by MickeySegal** [comment]
That was https://github.com/obophenotype/human-phenotype-ontology/issues/9915.  I don't know why this and some other findings are coming up twice.
